src/baseExplorer.ts

```diff
@@ -60,18 +60,21 @@ export class BaseExplorer {
             } else {
                 result = body;
             }
-            this.outputLine(label, `Message received from [${message.annotations["iothub-connection-device-id"]}]:`);
+            const timeMessage = message.enqueuedTimeUtc ? ` at ${message.enqueuedTimeUtc.toLocaleTimeString("en-US")}` : "";
+            this.outputLine(label, `Message received from [${message.annotations["iothub-connection-device-id"]}]${timeMessage}:`);

```

Comment:
put time in the front

Reply:
Updated. Thanks!

Reply:
![image](https://user-images.githubusercontent.com/1050213/44140935-3dd05a7c-a0ae-11e8-8028-effdeb916b44.png)